packages/std/src/sections.rs

```diff
@@ -0,0 +1,64 @@
+/// A sections decoder for the special case of two elements
+#[allow(dead_code)] // used in Wasm and tests only
+pub fn decode_sections2(data: Vec<u8>) -> (Vec<u8>, Vec<u8>) {
+    let section2_len: usize = if data.len() >= 4 {
+        u32::from_be_bytes([
```

Comment:
Why not encode with 2 bytes as now?
We expect/support > 64KB passed as a value for each section?

Although I guess it is a trivial cost to support this and allows us to never worry about size

Reply:
I'd rather not make unnecessary assumptions on data length.